DQMServices/FileIO/scripts/compareHistograms.py

```diff
@@ -170,10 +170,30 @@ def traverse_till_end(node, dirs_list, result, run_nr):
       for key in node.GetListOfKeys():
          traverse_till_end(key.ReadObj(), new_dir_list, result, run_nr)
    else:
-      path = tuple(new_dir_list)
-      if path not in get_blacklist(run_nr):
+      if not is_blacklisted(new_dir_list, run_nr):
+         path = tuple(new_dir_list)
          result[path] = node
 
+def get_node_name(node):
```

Comment:
@andrius-k not sure where this new method is used...

Reply:
@fabiocos Indeed, the invocation of this method was somehow lost but it has to be there. Just made another PR that reintroduces it: #27093 
Also updated the backport because it's not merged yet.